src/js/actions/DialogActions.js

```diff
@@ -0,0 +1,66 @@
+var AppDispatcher = require("../AppDispatcher");
+var DialogEvents = require("../events/DialogEvents");
+
+var DialogActions = {
+  alert: function (message) {
+    const dialogId = Symbol(message);
+    AppDispatcher.dispatchNext({
+      actionType: DialogEvents.ALERT,
+      message: message,
+      dialogId: dialogId
+    });
+    return dialogId;
+  },
+  alertClose: function (dialogId) {
+    AppDispatcher.dispatchNext({
+      actionType: DialogEvents.ALERT_CLOSE,
+      dialogId: dialogId
+    });
+  },
+  confirm: function (message) {
+    const dialogId = Symbol(message);
+    AppDispatcher.dispatchNext({
+      actionType: DialogEvents.CONFIRM,
```

Comment:
What do you think if we refactored `.CONFIRM`, `.PROMPT` and `.ALERT` action types to `.CONFIRM_SHOW`, `.PROMPT_SHOW` and `.ALERT_SHOW` for consistency? Like this our actions will all have one of these three suffixes: `SHOW`, `CLOSE`, `ACCEPT`.

And since we're at it, I would like to propose renaming `_CLOSE` to `_DISMISS` so that it's consistent with the store's function naming (cfr all the `onDismiss`, `dismissCallback` etc.). My point is that "close" doesn't express precisely the actual user interaction it means to represent: the user "dismisses" a dialog (including prompt/confirm interactions) when "canceling" it, while the dialogs are `close`d also upon a positive user interaction (for example when the user accepts a confirmation dialog). This would avoid any confusion to an hypothetical external observer regarding the `_CLOSE` events being triggered upon `_ACCEPT` actions.

@aldipower @philipnrmn